redirects.json

```diff
@@ -212,6 +212,30 @@
       "key": "/builders/get-started/networks/layer2/bobabeam/",
       "value": "/builders/get-started/networks/"
     },
+    {
+      "key": "/builders/integrations/wallets/",
+      "value": "/builders/integrations/"
+    },
+    {
+      "key": "/builders/integrations/wallets/index/",
+      "value": "/builders/integrations/"
```

Comment:
this can be removed, the one on lines 215-218 is the correct way to handle index pages